plugins/bed/src/index.ts

```diff
@@ -116,5 +117,17 @@ export default class BedPlugin extends Plugin {
         }
       },
     )
+
+    pluginManager.addToExtensionPoint(
+      'Core-guessTrackTypeForLocation',
+      (trackTypeGuesser: TrackTypeGuesser) => {
+        return (adapterName: string) => {
+          if (adapterName === 'BedpeAdapter') {
+            return 'VariantTrack'
```

Comment:
the choice to make it use VariantTrack may make it unable to use the linear arc display, which may be useful for BEDPE (arc connects the two regions). at one point you mentioned trying to add arc rendering for VariantTrack but that it was tricky. could revisit that perhaps...

Reply:
more tangential but multi-region arc may be useful too. the LinearAlignmentsArcDisplay ended up becoming multi-region capable recently, and similar concept could apply to the regular LinearArcDisplay